cpp/nvgraph/cpp/src/triangles_counting.cpp

```diff
@@ -58,7 +58,7 @@ TrianglesCount<IndexType>::~TrianglesCount()
 template <typename IndexType>
 void TrianglesCount<IndexType>::tcount_bsh()
 {
-    //printf("TrianglesCount: %s\n", __func__); fflush(stdout);
+    printf("TrianglesCount: %s\n", __func__); fflush(stdout);
     
```

Comment:
we should disable internal prints (also applies to line 90, 138, 179, 197)

Reply:
Will do. Thanks for catching that.

Reply:
Done.